src/ansi-c/ansi_c_entry_point.cpp

```diff
@@ -281,6 +281,8 @@ bool generate_ansi_c_start_function(
       }
 
       {
+        // TODO I have no idea what MAX refers to, or how the
+        // int_width-4 relates to it
         // assume argc is at most MAX/8-1
```

Comment:
I don't really understand this value either, but I'm wondering if this somehow relates to the (linux specific) kernel limit on argument size, which looks to be something like 1/4 of the maximum stack size - but of course that is total argument size, not the maximum *number* of arguments.

However, regardless of this particular magic, I thought that the bounds of {{{argc}}} was generally a platform specific value, rather than a language defined value (at least for C), hence, for example, POSIX has ARGS_MAX, so I don't thinking having a fixed value for this bound is necessarily the best thing to do. Might be worth raising an issue to cover sorting this out?